### ISSUE TITLE
Accessibility (Medium) - Accessible Texts for User Inputs (Guideline 2.5.3)

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/lsf#151]
**[Original report](https://bitbucket.org/laborstudents/lsf-flask/issue/151) by Hailey Barnett (Bitbucket: [barnetth](https://bitbucket.org/barnetth), GitHub: [barnetth](https://github.com/barnetth)).**

----------------------------------------

All fields should have accessible text to allow screen readers and other accessible devices to give information to those with disabilities. I have noticed a problem with selections on dropdowns where all the options are listed as either “blank” to the screen reader or nothing is stated at all. The screen reader can tell what the selections say when the mouse hovers over the selection, but not when the keyboard is being used. See the screenshot below for an example. This is mainly a problem on the “Labor Status Form” page, the “Manage Administrators” page, and the “Email Templates” page.

![](https://bitbucket.org/repo/ey5grgE/images/3341575354-Text_LI.jpg)
Please see this link \([https://www.w3.org/WAI/WCAG21/Techniques/general/G211](https://www.w3.org/WAI/WCAG21/Techniques/general/G211)\) for more information and examples.

If you need more information about this guideline, please visit the wiki \([http://172.31.2.178/accessibility/checklist](http://172.31.2.178/accessibility/checklist)\) or ask Hailey.

**Comment by bryantal** [comment]
**Original comment by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

PR is in. Waiting to be merged.
